doc/devel/contrib/rust-and-c++.md

```diff
@@ -1,6 +1,6 @@
 # Rust and C++
 
-Taskwarrior has historically been a C++ project, but as part of an [ongoing effort to replace its storage backend](https://github.com/GothenburgBitFactory/taskwarrior/issues/2770) it now includes a Rust library called TaskChampion.
+Taskwarrior has historically been a C++ project, but as part of an [effort to replace its storage backend](https://github.com/GothenburgBitFactory/taskwarrior/issues/2770) it now includes a Rust library called TaskChampion.
```

Comment:
```suggestion
Taskwarrior has historically been a C++ project, but as of taskwarrior-3.0.0, the storage backend is now provided by a Rust library called TaskChampion.
```

I doubt the link is very helpful any more and the language doesn't say that the storage backend was actually replaced but suggests that this is just being attempted with an "effort".